src/include/symbol_table/symbole_table.h

```diff
@@ -0,0 +1,101 @@
+#ifndef JUCC_SYMBOL_TABLE_SYMBOL_TABLE_H
+#define JUCC_SYMBOL_TABLE_SYMBOL_TABLE_H
+
+#include <iostream>
+#include <map>
+#include <string>
+#include <vector>
+namespace jucc {
+
+struct Node {
+  /**
+   * Used to store the name of the identifier
+   * obtained during tokenization
+   */
+  std::string identifier_;
+  /**
+   * Used to store the data type of the identifier
+   * One of int or float
+   */
+  std::string data_type_;
+  /**
+   * Used to store the nesting level of scoping
+   * Deeper the nesting, higher the scope
+   *
+   *      ...
+   *      int a0_;
+   *      if ( condition ){
```

Comment:
if ( condition ) {
Please add the space. Please 🥺🙏

Reply:
Space is there before and after "condition" 👀

Reply:
)     SPACE   {

Reply:
Gotcha!